wiki/Gameplay/Game_modifier/Hidden/en.md

```diff
@@ -63,7 +63,7 @@ The effects on difficulty for osu!catch regarding the use of the Hidden mod vary
 
 ### osu!mania
 
-In [osu!mania](/wiki/Game_mode/osu!mania), the Hidden mod functions as the inverse of the Fade In mod as the notes will fade out before you are supposed to hit them.
+In [osu!mania](/wiki/Game_mode/osu!mania), the Hidden mod functions as the inverse of the [Fade In](/wiki/Gameplay/Game_modifier/Fade_In) mod which, contrary to Fade In, notes fade in and fade out as they draw closer to the [judgement](/wiki/Gameplay/Judgement) line.
```

Comment:
this reads like an excessive edit, both the intended and final version (which also sounds like you've made a grammar mistake)